### ISSUE TITLE
Accessibility label detection fails in multi-language setups

### ISSUE THREAD

**Issue by d4rken** [d4rken-org/sdmaid-se#1344]
> Thanks

> [eu.darken.sdmse_logfile_1722496739146.zip](https://github.com/user-attachments/files/16453212/eu.darken.sdmse_logfile_1722496739146.zip)

@DRYN07 You did the exact opposite of what I asked :slightly_frowning_face: 

You didn't make any `chattr +i` modifications, the log shows that the AppCleaner loops. Looks like an issue with multiple system languages `en` and `hu`.

_Originally posted by @d4rken in https://github.com/d4rken-org/sdmaid-se/issues/1332#issuecomment-2262585330_

**Comment by d4rken** [comment]
@DRYN07 

* Did you use the Android 14 system feature of setting extra languages for specific apps?
* What is SD Maid's language and what is your system language?
* Does it work after a reboot?
* Does it work if you remove the second language?
* You are currently on `v1.1.2-rc0` does it work with the latest test version?

**Comment by DRYN07** [comment]
Sorry first I just wanted to make a log file because I am not really sure what is chattr +i modification and I wasn't sure if this was the issue in the first place (even if the problem is solved now, I would appreciate an answer about this)

Thank you very much your detailed help, the first one was the issue. I only use my system on my language, everything else is in English. Because I did not find an option to change it in the app, I suppose I changed id using this andorid feature.

Everything works fine now :))

- You are currently on v1.1.2-rc0 does it work with the latest test version?
The playstore does not show me newer version :/

**Comment by d4rken** [comment]
> Sorry first I just wanted to make a log file because I am not really sure what is chattr +i modification and I wasn't sure if this was the issue in the first place (even if the problem is solved now, I would appreciate an answer about this)

Using command line, the command [`chattr +i <file>`](https://stackoverflow.com/questions/48578108/what-the-differences-between-chattr-i-file-and-chmod-w-file) write protects a file. This is sometimes part of a technique to block the original app from re-creating specific files, e.g. specific caches.

> Thank you very much your detailed help, the first one was the issue. I only use my system on my language, everything else is in English. Because I did not find an option to change it in the app, I suppose I changed it using this Andorid feature.
> Everything works fine now :))

When testing this on my phone, this was not a problem, but sometimes I had to reboot the phone after changing the language. Does it now work because you removed the extra language, or did you reboot?


> > You are currently on v1.1.2-rc0 does it work with the latest test version?
> The Play Store does not show me newer version :/

To get the latest test version on Google Play you need to opt-in:

https://play.google.com/apps/testing/eu.darken.sdmse

**Comment by DRYN07** [comment]
> Using command line, the command [chattr +i <file>](https://stackoverflow.com/questions/48578108/what-the-differences-between-chattr-i-file-and-chmod-w-file) write protects a file. This is sometimes part of a technique to block the original app from re-creating specific files, e.g. specific caches.

Ohh I see now.

> When testing this on my phone, this was not a problem, but sometimes I had to reboot the phone after changing the language. Does it now work because you removed the extra language, or did you reboot?

I changed back the app language to use the system language in the android settings. I regularly reboot my phone (1-2x weekly), so that would solved it before

> To get the latest test version on Google Play you need to opt-in:

Hmm I was opted in before final release and I don't remember I opted out. But well, I did it again.
